planner/core/pb_to_plan.go

```diff
@@ -77,6 +77,10 @@ func (b *PBPlanBuilder) pbToTableScan(e *tipb.Executor) (PhysicalPlan, error) {
 	if !ok {
 		return nil, infoschema.ErrTableNotExists.GenWithStack("Table which ID = %d does not exist.", tblScan.TableId)
 	}
+	dbInfo, ok := b.is.SchemaByTable(tbl.Meta())
+	if !ok {
+		return nil, infoschema.ErrDatabaseNotExists.GenWithStack("Database of table ID = %d does not exist.", tblScan.TableId)
```

Comment:
```suggestion
		return nil, infoschema.ErrDatabaseNotExists.GenWithStack("Database which table ID = %d does not exist.", tblScan.TableId)
```

Reply:
The error is `database` not exist, not the table not exist...